massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -934,6 +951,14 @@ impl ProtocolWorker {
 
             // Check operation signature only if not already checked.
             if !self.checked_operations.contains_id(&operation_id) {
+
+                // Skip invalid Denunciation op
+                if !match &operation.content.op {
```

Comment:
I don't understand well this condition I think it need more doc. But as far as I understand what does happens if others operations are also invalid ? We will say pass but we should fail

Reply:
Here we are only checking if Denunciation operation are invalid, others operations will always be inserted in new operations no?